### PR TITLE
Fix arrogant moments in introduction

### PR DESCRIPTION
**Pull request by Amirka-Kh**

### CODE REVIEW

File: THESIS.md [hunk 1]
```diff
@@ -2,25 +2,25 @@
...
+Accurate project estimation and developer contribution analysis are critical to the success of software development projects. This research addresses the challenges of linking developer productivity metrics to project estimation, with the goal of improving project planning, tracking, and overall team efficiency. We analyzed a wide range of productivity metrics, identifying those that are most closely linked to project outcomes. Using this analysis, we developed a tool that analyzes developer performance based on GitHub repository data. The tool tracks individual contributions, links these statistics to project estimation, and includes a feature for predicting potential project cost overruns.
```
Comment (Timmmich19) [inline THESIS.md]: > Accurate project estimation and developer contribution analysis are critical to the success of software development projects.

Нашел подтверждение в introduction в статье https://www.mdpi.com/2073-431X/11/2/15 (статистика от standish group)

> We analyzed a wide range of productivity metrics, identifying those that are most closely linked to project outcomes.

Suggested:
We analyzed a wide range of models for estimating efforts and productivity metrics, identifying those that are most closely linked to project outcomes.

File: THESIS.md [hunk 2]
```diff
@@ -2,25 +2,25 @@
...
+In the fast-paced world of software development, the ability to accurately estimate project parameters and evaluate developer contributions is critical to a project’s success. Proper estimation helps set clear, achievable expectations for developers, which enhances their productivity and aligns their efforts with the project’s goals [1]. When developers see that their contributions are correctly recognized and rewarded, their motivation increases, leading to better work engagement. The relationship between work satisfaction and productivity is well-documented: when individuals satisfied with job and work environment and feel valued for the job they perform, they are more inclined to improve and grow, fostering a positive cycle of motivation and productivity [2].
```
Comment (Timmmich19) [inline THESIS.md:11]: > In the fast-paced world of software development, the ability to accurately estimate project parameters and evaluate developer contributions is critical to a project’s success.

Нашел подтверждение в introduction в 4 абзаце в статье https://sci-hub.sidesgame.com/10.1109/ACCESS.2020.3021664 (В особенности agile software development)

File: THESIS.md [hunk 3]
```diff
@@ -2,25 +2,25 @@
...
+Amazing results are achieved, when management have mastered the use of tools and metrics to set realistic expectations for both themselves and their development teams. These teams leverage their understanding of metrics to maintain high productivity and consistently meet project deadlines and budget constraints \cite{dmts1}. However, as these teams grow in size, they encounter new challenges \cite{dmts1}. Scaling these processes effectively becomes more difficult, and the complexity of tracking productivity across larger teams can lead to diminishing returns \cite{dmts1}.
 
 Despite the current state of project estimation and developer contribution analysis, there is substantial room for improvement. Our aim is not to critique the entire industry but to build on the foundation that already exists and push it forward. While there are teams that have mastered the art of project and productivity estimation, there is still a need for tools and frameworks that can help them maintain accuracy as they scale. Furthermore, many organizations are still in the process of mastering these techniques, and they too can benefit from a more structured approach to estimation.
```
Comment (Timmmich19) [inline THESIS.md:26]: Прочитал пару статей и пришел к более менее финальной версии третьего вопроса исследования:
RQ 3: How to estimate effort based on project and team data?
  RQ 3.1: What models/techniques are available and on what data is estimation based?
  RQ 3.2: What factors influence to accuracy of effort estimation?
  RQ 3.3: How to improve effort estimation?

File: THESIS.md [hunk 4]
```diff
@@ -2,25 +2,25 @@
...
+The practice of estimating project timelines and budgets, as well as measuring developer productivity, has evolved significantly over time. Traditionally, project estimation involved basic metrics like the number of lines of code (LOC), budget allocation, and deadline setting [3]. While these methods provided a foundation, they often fell short of capturing the full complexity of modern software development environments. Similarly, developer contribution was historically measured by lines of code produced, hours and the number of bugs found\fixed, failing to account for the qualitative aspects of a developer’s work, such as innovation, collaboration, or code quality [2, 3].
```
Comment (Timmmich19) [inline THESIS.md]: > While these methods provided a foundation, they often fell short of capturing the full complexity of modern software development environments.

Suggested:
Although these methods laid the foundation, they often failed to capture the full complexity of modern software development environments, especially those that followed the agile methodology.

(https://sci-hub.sidesgame.com/10.1109/ACCESS.2020.3021664) 3rd page

File: THESIS.md [hunk 5]
```diff
@@ -2,25 +2,25 @@
...
+Amazing results are achieved, when management have mastered the use of tools and metrics to set realistic expectations for both themselves and their development teams. These teams leverage their understanding of metrics to maintain high productivity and consistently meet project deadlines and budget constraints \cite{dmts1}. However, as these teams grow in size, they encounter new challenges \cite{dmts1}. Scaling these processes effectively becomes more difficult, and the complexity of tracking productivity across larger teams can lead to diminishing returns \cite{dmts1}.
 
 Despite the current state of project estimation and developer contribution analysis, there is substantial room for improvement. Our aim is not to critique the entire industry but to build on the foundation that already exists and push it forward. While there are teams that have mastered the art of project and productivity estimation, there is still a need for tools and frameworks that can help them maintain accuracy as they scale. Furthermore, many organizations are still in the process of mastering these techniques, and they too can benefit from a more structured approach to estimation.
```
Comment (Timmmich19) [inline THESIS.md]: > While there are teams that have mastered the art of project and productivity estimation, there is still a need for tools and frameworks that can help them maintain accuracy as they scale.

Suggested:
Although there are teams that have mastered the art of estimating projects and performance, there is still a need for tools and frameworks that will help them maintain accuracy as they scale, as well as help automate estimation processes to maintain their relevance compared to other important project management tasks.

File: THESIS.md (hunk 1 above)
Comment (NikRam822) [inline THESIS.md]: _Accurate project estimation and developer contribution analysis are critical to the success of software development projects._   - Need a reference

File: THESIS.md [hunk 6]
```diff
@@ -2,25 +2,25 @@
...
 
 Our tool was tested in a real-world setting by a software development company. The results were encouraging: the tool identified a developer who claimed to be overworked but whose contributions were average compared to the rest of the team. The situation led to the developer's voluntary departure, which, according to the company, had a positive impact on team morale and overall productivity. These findings demonstrate the potential of our tool to improve both project estimation and team dynamics, ultimately contributing to more efficient project management.
```
Comment (NikRam822) [inline THESIS.md]: Стоит добавить самокритику. Типо, что это лишь один кейс и для серьезных выводов требуется исследование.

File: THESIS.md [hunk 7]
```diff
@@ -2,25 +2,25 @@
...
 
-Moreover, many project managers hold the mistaken belief that there is a set of universal metrics that can comprehensively assess all aspects of productivity and project complexity \cite{dmts1}. This reliance on oversimplified or misunderstood metrics contributes to the perpetuation of common myths in project management, which we will address in the next chapter. These misconceptions, coupled with incomplete data, result in suboptimal decision-making. Consequently, management often uses these flawed metrics to estimate future project parameters, compounding the problem \cite{dmts1}.
+Moreover, having valid data is challenging, as collecting metrics can be difficult. Some managers tend to rely on only one or two metrics to base their decisions. While this approach is better than having no metrics at all, it must be used correctly. For instance, if a manager tracks the number of tasks accomplished by developers, all tasks should have similar scope in terms of time and complexity, which can be achieved by breaking down larger tasks into smaller ones. So, manager should also set right development process and make right calculations.
```
Comment (NikRam822) [inline THESIS.md]: _Moreover, having valid data is challenging, as collecting metrics can be difficult. Some managers tend to rely on only one or two metrics to base their decisions. While this approach is better than having no metrics at all, it must be used correctly._ - Need a reference

File: THESIS.md [hunk 8]
```diff
@@ -2,25 +2,25 @@
...
 
-That said, not all is bleak. There are mature teams and experienced managers who have mastered the use of tools and metrics to set realistic expectations for both themselves and their development teams. These teams leverage their understanding of metrics to maintain high productivity and consistently meet project deadlines and budget constraints \cite{dmts1}. However, as these teams grow in size, they encounter new challenges \cite{dmts1}. Scaling these processes effectively becomes more difficult, and the complexity of tracking productivity across larger teams can lead to diminishing returns \cite{dmts1}.
+Amazing results are achieved, when management have mastered the use of tools and metrics to set realistic expectations for both themselves and their development teams. These teams leverage their understanding of metrics to maintain high productivity and consistently meet project deadlines and budget constraints \cite{dmts1}. However, as these teams grow in size, they encounter new challenges \cite{dmts1}. Scaling these processes effectively becomes more difficult, and the complexity of tracking productivity across larger teams can lead to diminishing returns \cite{dmts1}.
```
Comment (NikRam822) [inline THESIS.md]: _Amazing results are achieved, when management have mastered the use of tools and metrics to set realistic expectations for both themselves and their development teams. These teams leverage their understanding of metrics to maintain high productivity and consistently meet project deadlines and budget constraints \cite{dmts1}._ - Need a reference

 _However, as these teams grow in size, they encounter new challenges \cite{dmts1}. Scaling these processes effectively becomes more difficult, and the complexity of tracking productivity across larger teams can lead to diminishing returns \cite{dmts1}._ - Need a reference

File: THESIS.md (hunk 5 above)
Comment (NikRam822) [inline THESIS.md]: _Furthermore, many organizations are still in the process of mastering these techniques, and they too can benefit from a more structured approach to estimation._ - Need a reference

File: THESIS.md [hunk 9]
```diff
@@ -85,4 +85,15 @@ Once we have collected the relevant metrics, the next step is to use statistical
 
 In this chapter, we have explored the common myths and mistakes associated with productivity and project estimation, and proposed a more comprehensive approach to addressing these issues. By collecting the right metrics and using a multidimensional approach, we can better understand the productivity of development teams and make more accurate project estimations. In the next chapter, we will delve into the practical details of implementing this approach, including the challenges of collecting reliable metrics and using them effectively in project planning.
```
Comment (NikRam822) [inline THESIS.md:89]: _By collecting the right metrics and using a multidimensional approach, we can better understand the productivity of development teams and make more accurate project estimations_ - Need a reference

File: THESIS.md (hunk 6 above)
Comment (NikRam822) [reply THESIS.md]: Или можно сказать, что в первом приближении есть успешный кейс использования инструмента

File: THESIS.md [hunk 10]
```diff
@@ -85,4 +85,15 @@ Once we have collected the relevant metrics, the next step is to use statistical
...
 
+References:
```
Comment (NikRam822) [inline THESIS.md:91]: Я не уверен, что это правильный формат цитирования

File: THESIS.md (hunk 1 above)
Comment (NikRam822) [reply THESIS.md]: Решение:

Текст меняем : We analyzed a wide range of models for estimating efforts and productivity metrics, identifying those that are most closely linked to project outcomes.

Референс в интро.

File: THESIS.md (hunk 6 above)
Comment (NikRam822) [reply THESIS.md]: Решение:
Переформулировать и уточнить, что ДЛЯ НАШЕГО КЕЙСА это сработало

File: THESIS.md (hunk 2 above)
Comment (NikRam822) [reply THESIS.md:11]: Решение:

Добавлеям в рефернс 1 рефернес Тимура

File: THESIS.md (hunk 4 above)
Comment (NikRam822) [reply THESIS.md]: Решение: 
Добавлеям в рефернс 2,3 рефернес Тимура

File: THESIS.md (hunk 7 above)
Comment (NikRam822) [reply THESIS.md]: Решение:

Найти референс и вставить его

File: THESIS.md (hunk 8 above)
Comment (NikRam822) [reply THESIS.md]: Решение:

Нати референсы и вставить их

File: THESIS.md (hunk 5 above)
Comment (NikRam822) [reply THESIS.md]: Решение:

1) Переформулировать более точно про масштабируемость (as they scale). Также добавить инфу о том, что нужны фреймворки чтобы не допускать ошибок при использовании метрик а также нужны инструменты  для автоматизации работы с метриками.
2) Найти и добавить референсы подтверждающие тезисы выше.

File: THESIS.md (hunk 3 above)
Comment (NikRam822) [reply THESIS.md:26]: Решение:
Переформулировать вопросы для RQ2 и RQ3( коммент Тимура)

File: THESIS.md (hunk 9 above)
Comment (NikRam822) [reply THESIS.md:89]: Пока что забиваем, т.к. это не интро

File: THESIS.md (hunk 10 above)
Comment (NikRam822) [reply THESIS.md:91]: Решение:
Меням формат цитирования. (в последнюю очередь делаем это)

File: THESIS.md [hunk 11]
```diff
@@ -106,3 +106,13 @@ References:
...
+
+[18] Tapasco-Alzate, Omar, Jaime A. Giraldo-García and Diógenes de Jesus Ramírez-Ramírez. “Productivity metrics in the context of knowledge work: literature vs practice.” International Journal of Productivity and Performance Management ahead-of-print (2021): n. pag.
+
+[19] A. Coskun and R. Erturgut, “An empirical research on developing a logistics performance scale,” International Journal of Productivity and Performance Management, vol. ahead-of-print, Feb. 2021. doi: 10.1108/ijppm-10-2020-0513.
+
+[20] Schmidt, Roy & Lyytinen, Kalle & Keil, Mark & Cule, Paul. (2001). Identifying Software Project Risks: An International Delphi Study. J. of Management Information Systems. 17. 5-36. 10.1080/07421222.2001.11045662.
+
+[21] Boehm, B. W., & Turner, R. (2004). "Balancing Agility and Discipline: A Guide for the Perplexed."
```
Comment (Amirka-Kh) [inline THESIS.md]: Should be replaced with `Boehm, B. (2004). Balancing Agility and Discipline: A Guide for the Perplexed. In: Ramamoorthy, C.V., Lee, R., Lee, K.W. (eds) Software Engineering Research and Applications. SERA 2003. Lecture Notes in Computer Science, vol 3026. Springer, Berlin, Heidelberg. https://doi.org/10.1007/978-3-540-24675-6_1`

**Review by Amirka-Kh — COMMENTED** [review]
`+ [21] 


Should be replaced with `Boehm, B. (2004). Balancing Agility and Discipline: A Guide for the Perplexed. In: Ramamoorthy, C.V., Lee, R., Lee, K.W. (eds) Software Engineering Research and Applications. SERA 2003. Lecture Notes in Computer Science, vol 3026. Springer, Berlin, Heidelberg. https://doi.org/10.1007/978-3-540-24675-6_1`

**Pull request merged**